includes/services.inc.php

```diff
@@ -129,7 +129,7 @@ function poll_service($service)
 
     // If we do not have a cmd from the check script, build one.
     if ($check_cmd == "") {
-        $check_cmd = $config['nagios_plugins'] . "/check_" . $service['service_type'] . " -H " . ($service['service_ip'] ? $service['service_ip'] : $service['hostname']);
```

Comment:
Hello!
I added 
$config['force_ip_to_sysname'] = true;
to file
/master/html/includes/table/alerts.inc.php
It works fine! 
Thanks!